rcl/src/rcl/context.c

```diff
@@ -103,9 +103,10 @@ rcl_context_get_rmw_context(rcl_context_t * context)
   return &(context->impl->rmw_context);
 }
 
-void
+rcl_ret_t
 __cleanup_context(rcl_context_t * context)
 {
+  rcl_ret_t ret = RCL_RET_OK;
```

Comment:
This introduces a new variable that will cause scoped variables to shadow this one. See line 115.

Reply:
Whoops, I accidentally omitted line 115.

Reply:
See https://github.com/ros2/rcl/pull/763/commits/6f3cab1371766922ad1422d03f1259766c11d417,